include/librealsense2/h/rs_aus.h

```diff
@@ -96,6 +96,12 @@ extern "C" {
     */
     const char* rs2_aus_get_counter_data(const rs2_strings_list* buffer, int i, rs2_error** error);
 
+    /**
+    * Get aus information as raw_data_buffer
+    * \param[out] error   if non-null, receives any error that occurs during this call, otherwise, errors are ignored
+    * \return   rs2_raw_data_buffer * with aus information which should be released by rs2_delete_raw_data
+    */
+    const rs2_raw_data_buffer * rs2_aus_receive_json_data(rs2_error ** error);
```

Comment:
I think  "rs2_aus_get_json_data" or even "rs2_aus_get_data" are better choices.
@Nir-Az what do you think ?

Reply:
Agree, your call :)

Reply:
Changed